backend/src/main/java/ai/verta/modeldb/ModelDBConstants.java

```diff
@@ -13,7 +13,6 @@
   String CLOUD_BUCKET_NAME = "cloudBucketName";
   String MINIO_ENDPOINT = "minioEndpoint";
   String AWS_REGION = "aws_region";
-  String DATABASE = "database";
```

Comment:
We should have to remove other DB config parameters which is not used now like 
```
  String DB_TYPE = "DBType";
  String MIN_CONNECTION_POOL_SIZE = "minConnectionPoolSize";
  String MAX_CONNECTION_POOL_SIZE = "maxConnectionPoolSize";
  String CONNECTION_TIMEOUT = "connectionTimeout";
```

Reply:
I'll do that in a follow-up. I'm not a big fan of centralized constants list for things that are local. That's why we missed that some feature flags were not being used at all, I believe.

Reply:
Ok. agree